README.md

```diff
@@ -1,70 +1,138 @@
-# Getting Started with Create React App
+# 📗 Table of Contents
```

Comment:
- [ ] OPTIONAL: Kindly provide a note on your readme pointing out that only new cars generated by users can be deleted.  The ones that come with the app cannot be deleted.  

That way, we can avoid future confusion. 👍